contrib/vdsm-dev

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env bash
+# Author: Albert Esteve <aesteve@redhat.com>
+
+venv_path=$HOME/.venv/vdsm
+venv_activate_path=${venv_path}/bin/activate
+user_storage_path=/var/tmp/vdsm-storage/
+
+print_help()
+{
+    echo "oVirt/vdsm command line helper."
+    echo "Should be run from the repository root folder."
+    echo "Usage: $0 CMD [options]"
+    echo
+    echo "Available commands:"
+    echo "help:          Print this message and exits."
+    echo "build:         Build vdsm and make RPMs."
+    echo "build-install: Build vdsm, make RPMs, and update local packages."
+    echo "test:          Run any test from specified environment using tox."
+    echo "test-storage:  Run one/all storage tests using tox."
+    echo "show-coverage: Show coverage results generated after a test run."
+    echo "               Receives an environment with '-e env'. [default=storage]"
+}
+
+venv_create()
+{
+    echo "Creating venv at $venv_path"
+    python3 -m venv ${venv_path}
+    source "${venv_activate_path}"
+    pip install --upgrade pip
+    pip install -r docker/requirements.txt
+    deactivate
+}
+
+make_storage()
+{
+    if [ ! -d $user_storage_path ]; then
+        echo "Make storage at $user_storage_path"
+        make storage
+    fi
+}
+
+test_path()
+{
+    path=$1
+    storage_prefix=storage/
+    if [ ! -f "tests/$path" ]; then
+        path=${storage_prefix}${path}
+        if [ ! -f "tests/$path" ]; then
+            echo "No test $path found"
+            exit 1
+        fi
+    fi
+}
```

Comment:
This is... something I thought it may be nice to have but then was too complex for what I was using it, so I will probably drop it.

The idea was to check if the test file exists, and if not, look for it and give the correct path to `tox`. And if it doesn't, just show an error before executing the test. Maybe with a `find` command, this could be achieved nicely.

This way one could do:
```
vdsm-dev test-storage lvm.py
```
And get the correct test without having to worry about the exact location of the file. But the gain is relatively small and we have to be careful in case the same test filename exists in two environments. So again, probably will drop it.